reconcile/ocm_cluster_admin.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import logging
+from typing import Any, Mapping
+
+from reconcile import queries
+from reconcile.status import ExitCodes
+from reconcile.utils.ocm import OCMMap
+from reconcile.ocm.utils import cluster_disabled_integrations
+
+QONTRACT_INTEGRATION = "ocm-cluster-admin"
+
+
+def _cluster_is_compatible(cluster: Mapping[str, Any]) -> bool:
```

Comment:
NIT: regarding the name, compatible with what?

Reply:
take a look at the rest of the OCM integrations